README.md

```diff
@@ -6,6 +6,15 @@ The `Semantic Model Generator` is used to generate a semantic model for use in y
 
 We currently leverage credentials saved as environment variables. Note, `host` is optional depending on your Snowflake deployment.
 
+A. To find your Account Name, [follow these instructions](https://docs.snowflake.com/en/sql-reference/functions/current_account_name), or execute the following sql command in your account.
+
+```sql
+SELECT CURRENT_ACCOUNT_NAME()
+```
+
+B. To find the host for your account, [follow these instructions](https://docs.snowflake.com/en/user-guide/organizations-connect#connecting-with-a-url).
```

Comment:
still not super clear to me what the host should be, the documentation isn't super clear 😢   (the doc's saying just be `.snowflakecomputing.com` for most cases?)

Reply:
yes - I'll add here as well.